node/bft/examples/simple_node.rs

```diff
@@ -171,14 +171,14 @@ pub async fn start_primary(
         BatchHeader::<CurrentNetwork>::MAX_GC_ROUNDS as u64,
     );
     // Initialize the gateway IP and dev mode.
-    let (ip, dev) = match peers.get(&node_id) {
-        Some(ip) => (Some(*ip), None),
-        None => (None, Some(node_id)),
+    let (ip, storage_mode) = match peers.get(&node_id) {
+        Some(ip) => (Some(*ip), StorageMode::Production),
```

Comment:
same principle: we wouldn't want to write to the production storage when running an example; that being said, that depends on the purpose of the example, and I'm not sure what it currently is